### PR TITLE
Query: Enables ThenInclude for full multi-level include support.

### PR DESCRIPTION
**Pull request by anpete**

### CODE REVIEW

**Comment by anpete** [comment]
cc @maumar

File: src/EntityFramework.Core/Extensions/QueryableExtensions.cs [hunk 1]
```diff
@@ -1298,9 +1299,10 @@ public static class QueryableExtensions
 
         public static readonly MethodInfo IncludeMethodInfo
             = typeof(QueryableExtensions)
-                .GetTypeInfo().GetDeclaredMethod("Include");
+                .GetTypeInfo().GetDeclaredMethods("Include")
+                .Single(mi => mi.GetParameters().Any(pi => pi.Name == "navigationPropertyPath"));
```
Comment (bricelam) [inline src/EntityFramework.Core/Extensions/QueryableExtensions.cs]: Should these be matched by parameter type instead?

Comment (anpete) [inline src/EntityFramework.Core/Extensions/QueryableExtensions.cs]: This seemed easier/simpler.

**Comment by bricelam** [comment]
:shipit:

**Pull request closed without merging**